tests/console/nftables.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: nftables
+# Summary: Test nftables with firewalld
+# Maintainer: Zaoliang Luo <zluo@suse.de>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # check nftables, stop iptables and start firewalld by need
+    my $logs = "/tmp/log";
+
+    #check that nftables has been already installed
+    assert_script_run('echo -e "____ nftables version and rules ____\n\n" > ' . $logs);
+    assert_script_run("rpm -qa nftables |& tee -a $logs");
```

Comment:
I guess you're tring to get the version here, please use: zypper_call('info nftables'); instead

Reply:
actually this is almost the same what I want to check: build information should be there for logs only if something goes wrong and we know which version.

Reply:
Please use the suggested `zypper_call('info nftables');`